arlas-tagger-core/src/main/java/io/arlas/tagger/service/TagExecService.java

```diff
@@ -93,6 +93,14 @@ public void processRecords(ConsumerRecords<String, String> records) {
                     updatedTotal += opUpdateResponse.updated;
                     UpdateResponse tagUpdateResponse = TaggingStatus.getInstance().updateStatus(tagRequest, opUpdateResponse, true, statusTimeout);
                     LOGGER.trace("Tagged {} documents [total={} / {}%] (failed={}) with processtime={}ms", opUpdateResponse.updated, tagUpdateResponse.updated, tagUpdateResponse.progress, tagUpdateResponse.failed, (System.currentTimeMillis() - t0));
+                    if (tagUpdateResponse.failed > 0) {
+                        int newErrors =  (int)tagUpdateResponse.failed - nbErrors;
+                        for(int i = nbErrors; i < nbErrors + newErrors; i++) {
+                            UpdateResponse.Failure f = tagUpdateResponse.failures.get(i);
+                            LOGGER.error("Failure {}: \n - id : {} \n - type : {} \n - message : {}", i, f.id, f.type, f.message);
```

Comment:
I believe the context should also be printed (`"Tagged {} documents [total={} / {}%] (failed={}) with processtime={}ms"  ...`) before the loop in order to understand better the perimeter of the failure.

Reply:
It's already printed just beofre the `if (tagUpdateResponse.failed > 0)` no ?

Reply:
Fixed